docs/getstarted/educators-and-students.md

```diff
@@ -0,0 +1,152 @@
+
+When you share a collaborative session, the person you're working with sees the context of the workspace in their editor. This means your classmate can read the code you shared without having to clone a repo or install any dependencies your code relies on. They can help you with your code in the Visual Studio Code environment that's familiar to them.
+
+Each of you can open files, navigate, edit code, or highlight - and changes are instantly reflected. As you edit you can see your classmate's cursor, jump to the same location, and follow their actions.
+
+You can also debug together using VS Code's debugging features, like hovers, locals and watches, the stack trace or the debug console. You are both able to set breakpoints and advance the debug cursor to step through the session.
+
+For more details about what you can do with Live Share, visit the [how-to-guide](https://learn.microsoft.com/visualstudio/liveshare/use/install-live-share-visual-studio-code).
+
+### Get started with Live Share
+
+To get started with using Live Share in VS Code, you'll need to download the [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare) extension from the VS Code Marketplace.
+
+![Live Share extension](images/educators-and-students/live-share-extension.png)
+
+This extension includes everything you need to start collaboratively editing and debugging in real time. This provides you and your team/class with a one-step installation, in order to begin pair programming, performing remote code reviews, driving interactive lectures, and more, without needing to leave Visual Studio Code.
+
+Once you log into your GitHub account, you can see the Live Share icon in the Activity Bar.
+
+![Live Share icon in the Activity Bar](images/educators-and-students/liveshare-icon.png)
+
+#### Starting a Live Share session
+
+If you select **Start Collaboration session** from the Session Details menu, an invitation link to your session is automatically copied to your clipboard. You can share this link with anyone you'd like to collaborate with, as long as they also have VS Code and the Live Share extension installed.
+
+![Live Share invitation](images/educators-and-students/liveshare-invitation.png)
+
+When your classmate joins the session, you get a notification, and see their name come up under Participants.
+
+![Live Share joined](images/educators-and-students/liveshare-joined.png)
+
+#### Joining a Live Share session
+
+If you select **Join Collaboration session** from the Session Details menu, you're able to enter the URL you received.
+
+![Joining a Live Share session](images/educators-and-students/liveshare-join-session.png)
+
+You should be able to see and interact with your classmate's code on your screen!
+
+## Free student lessons
+
+### Get started with VS Code
```

Comment:
If we do choose to update the above header to `## Student resources`, these next few could be renamed to also be consistent, like `### Lessons for...` or `### Getting started with VS Code lessons`